Formula/q/qt-base.rb

```diff
@@ -0,0 +1,263 @@
+class QtBase < Formula
```

Comment:
`qtbase` could be better to align with repo name. At least avoids the odd naming when we hit `qt5compat` and deciding if whitespaces in names like "Qt Serial Bus" matter.